src/Illuminate/Pagination/Cursor.php

```diff
@@ -119,7 +120,7 @@ public static function fromEncoded($encodedString)
             return null;
         }
 
-        $parameters = json_decode(base64_decode(str_replace(['-', '_'], ['+', '/'], $encodedString)), true);
+        $parameters = json_decode(Str::fromUrlSafeBase64($encodedString), true);
```

Comment:
Whoa! These two are quite a mouthful. Maybe, this should be split into just the replace part and could be manually wrapped inside `Str::fromBase64()`/`Str::toBase64()`